docs/Anjay_integrations/RaspberryPi_integration/Implementing_LwM2M_objects_on_RaspberryPi.md

```diff
@@ -0,0 +1,490 @@
+
+## Step 3: Register your device in Coiote DM
+
+0. Disable the Svetovid service:
+
+    `sudo systemctl disable svetovid.service --now`
+
+0. Set the server connection details:
+
+	 - in `/etc/svetovid/config/security.json`:
+			```
+				{
+					"1":
+					{
+				    	"ssid": "1",
+				    	"server_uri": "coaps://eu.iot.avsystem.cloud:5684",
+				    	"is_bootstrap": "0",
+				    	"security_mode": "psk",
+				    	"pubkey_or_identity_hex": "XXXX",
+				    	"privkey_or_psk_hex": "XXXX",
+				    	"server_pubkey_hex": "",
+				    	"holdoff_s": "0",
+				    	"bs_timeout_s": "0"
+					}
+				}
+			```
+
+		- in `/etc/svetovid/config/svd.json`:
+			```
+				{
+					"device": {
+				    	"endpoint_name": "YYYY",
+				    	"udp_listen_port": 1234
+					},
+					"logging": {
+				    	"default_log_level": "trace",
+				    	"log_level": {
+				        	"svd": "trace"
+				    	}
+					},
+					"lwm2m_version_config": {
+				    	"min": "1.0",
+				    	"max": "1.1"
+					},
+					"in_buffer_size_b": 1024,
+					"out_buffer_size_b": 1024,
+					"msg_cache_size_b": 65536
+				}
+			```
+
+    !!! note
+        XXXX and YYYY should be replaced by the chosen values of endpoint name, PSK identity and PSK key.
+
+    !!! tip
+        XXXX values are in hex. To convert your plain text, use the following:
+            ```echo -n '<your-text-value>' | xxd -p ```
+
+0. Restart the Svetovid service:
+
+    `sudo systemctl start svetovid.service --now`
+
+## Step 4: Connecting to the LwM2M Server
+
+ To connect to Coiote IoT Device Management LwM2M Server, please register at [https://eu.iot.avsystem.cloud](https://eu.iot.avsystem.cloud).
+
+ To connect the board:
+
+ 1. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+ 2. In **Device Inventory**, click **Add device**.
+ 3. Select the **Connect your LwM2M device directly via the Management server** tile.
+       ![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+    3. In the **Device credentials** step:
+         - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
+             ![Device credentials step](images/raspberry_pi_test_credentials.png "Device credentials step")
+         - In the **Security mode** section, select the security mode you've choosen in [Compile the application](#compile-the-application) step:
+
+            !!! note
+                If you choose **Pre-Shared Key** mode then type:
+
+                - In the **Key identity** field, the same name as in the Endpoint name field.
+                - In the **Key** field, the shared secret used in the device-server authentication.
+
+    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+
+ 0. Analyze the objects exposed by the device.
+    ![Device objects](images/rasp.png "Objects")
+
+!!! note
+    Your RaspberryPi-based device will feature a number of default LwM2M objects provided by Svetovid, for instance the temperature object `/3303`.
```

Comment:
There is no `/3303` object on the screenshot (it is added later in the article). Maybe change it to the `3311 Light Control`?